include/vcpkg/vcpkgcmdarguments.h

```diff
@@ -335,7 +335,7 @@ namespace vcpkg
         std::string command;
 
         Optional<StringLiteral> m_detected_ci_environment_name;
-        CIKind m_detected_ci_environment_type;
+        CIKind m_detected_ci_environment_type = CIKind::None;
```

Comment:
I believe this is the only actual change here and is already in https://github.com/microsoft/vcpkg-tool/commit/cfd44ca645aa937f5326b9cfb2aa1e56106d0ce4

Reply:
I wounder why it is then still visualized as changed line

Reply:
It uses the merge-base of the source and the target for the "red" side, not the target itself.